DatadogSessionReplay/Sources/Recorder/Utilities/SwfitUIImage+SessionReplay.swift

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+#if os(iOS)
+import SwiftUI
+
+/// Mapping SwiftUI orientation to UIImage orientation
+@available(iOS 13.0, tvOS 13.0, *)
+internal extension SwiftUI.Image.Orientation {
```

Comment:
/suggestion Can we move this to `SwiftUI/Image.swift` instead? to limit file addition for such simple conversion.

/suggestion-2 My personal preference here is to use an `init` to have a more idiomatic casting pattern.
```swift
extension UIImage.Orientation {
	init(_ orientation: SwiftUI.Image.Orientation) {
		...
	}
}
```